website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -72,8 +72,8 @@ until the LTS release reaches its end of maintenance.
 
 | Consul Version | Compatible Envoy Versions                                                          |
 | -------------- | -----------------------------------------------------------------------------------|
-| 1.18.x Ent     | 1.29.4, 1.28.3, 1.27.5, 1.26.8, 1.25.11                                            |
-| 1.15.x Ent     | 1.29.4, 1.27.5, 1.26.8, 1.25.11, 1.24.12, 1.23.12, 1.22.11                         |
+| 1.18.x Ent     | 1.29.x, 1.28.x, 1.27.x, 1.26.x, 1.25.x                                             |
+| 1.15.x Ent     | 1.29.x, 1.28.x, 1.27.x, 1.26.x, 1.25.x, 1.24.x, 1.23.x, 1.22.x                     |
```

Comment:
Noticed today that 1.28 had also been left out of 1.15's support list, so adding here